src/routes/(auth)/connexion/+page.svelte

```diff
@@ -11,22 +12,26 @@
 <div class="container login">
 	<form method="POST" use:enhance>
 		<div class="form-widget">
-			<h1 class="header">Me connecter</h1>
+			<h1 class="header">J'ai déjà un compte</h1>
```

Comment:
Titles have no styles (because of the reset.css added recently) for both inscriptions and connexion pages.